frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerRightTabs.tsx

```diff
@@ -35,35 +24,47 @@ const PipelineRunDrawerRightTabs: React.FC<PipelineRunDrawerRightTabsProps> = ({
   task,
   executions,
 }) => {
-  const [selection, setSelection] = React.useState(PipelineRunNodeTabs.INPUT_OUTPUT);
+  const [selection, setSelection] = React.useState<string>(PipelineRunNodeTab.INPUT_OUTPUT);
```

Comment:
What if `PipelineRunNodeTab.INPUT_OUTPUT` tab is disabled here and it's set as the default selection tab?

Reply:
Added a fallback to default to the "Details" tab when no inputs/outputs exist.